.changes/next-release/Bug Fix-a8d6edab-136e-4a0e-ba04-1bdd4e1a45eb.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Should delete this since its non user facing?

Reply:
yeah I'd say so. Also would suggest using "telemetry" as the commit-message prefix (aka topic)